ntbs-service/Program.cs

```diff
@@ -1,152 +1,837 @@
 ﻿using System;
 using System.Linq;
 using EFAuditer;
-using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.EntityFrameworkCore;
 using Microsoft.Extensions.DependencyInjection;
+using ntbs_service;
 using ntbs_service.DataAccess;
 using ntbs_service.Models;
 using Sentry;
 using Serilog;
 using Serilog.Events;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Security.Claims;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+using Hangfire;
+using Hangfire.Console;
+using Hangfire.SqlServer;
+using Microsoft.ApplicationInsights.AspNetCore;
+using Microsoft.ApplicationInsights.AspNetCore.Extensions;
+using Microsoft.ApplicationInsights.DependencyCollector;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.Authentication.OpenIdConnect;
+using Microsoft.AspNetCore.Authentication.WsFederation;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.DataProtection;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.HttpOverrides;
+using Microsoft.AspNetCore.Mvc.Authorization;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Hosting;
+using ntbs_service.Authentication;
+using ntbs_service.DataMigration;
+using ntbs_service.Helpers;
+using ntbs_service.Jobs;
+using ntbs_service.Middleware;
+using ntbs_service.Models.Entities;
+using ntbs_service.Properties;
+using ntbs_service.Services;
+using ZNetCS.AspNetCore.Authentication.Basic;
+using ZNetCS.AspNetCore.Authentication.Basic.Events;
+using Constants = ntbs_service.Constants;
+using WebApplication = Microsoft.AspNetCore.Builder.WebApplication;
 
-namespace ntbs_service
+
+SetUpLogger();
+try
+{
+    Log.Information("Building web app");
+    
+    var builder = WebApplication.CreateBuilder(args);
+    ConfigureHost(builder);
+    ConfigureServices(builder);
+    var app = builder.Build();
+    Configure(app);
```

Comment:
This section of code is the main change with the switch to building a `WebApplication`. We now have no startup file, everything is now here in `Program`. 
I've basically copied/pasted the startup file into this one and then called the `Configure` methods when needed.